src/hotspot/share/classfile/bytecodeAssembler.hpp

```diff
@@ -155,7 +159,7 @@ class BytecodeConstantPool : ResourceObj {
 
   u2 string(Symbol* str, TRAPS) {
     u2 utf8_entry = utf8(str, CHECK_0);
-    return find_or_add(BytecodeCPEntry::string(utf8_entry), THREAD);
```

Comment:
So `utf8_entry` is now unneeded.

Reply:
I added some comment to make this make more sense.  It needs the Symbol from the original constant pool to compare, since the utf8 constant pool entry from the one we're copying doesn't point to the UTF8 entry anymore (it points to the symbol).  Took me a while to figure out what hash is, so I renamed it value.